src/components/StudentHome.jsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
+import { Row, Col, Alert, Glyphicon } from 'react-bootstrap';
 import WelcomeMessage from './WelcomeMessage';
 import PlanOfStudy from './PlanOfStudy';
+import NavPanel from './NavPanel';
 
 // The wrapper for all components on a student home page
 const StudentHome = () => (
-  <div>
-    <h2>
-      Student Home
-    </h2>
-    <WelcomeMessage student="Ariana Olson" />
-    <PlanOfStudy />
-  </div>
+  <Row>
```

Comment:
Can we get some comments up in here? I wanna know what all the Col attributes are, and I'm not sure what "do yer course plan" is doing.